.chaostest/README.md

```diff
@@ -0,0 +1,68 @@
+chaostest
```

Comment:
To reduce the amount of top level directories, I would suggest moving the `chaostest` project into `.maintain`. The *chaostest* project will help us *maintain* the *Substrate* project.